tests/test_agent.py

```diff
@@ -1,87 +0,0 @@
-# -*- coding: utf-8 -*-
-# ------------------------------------------------------------------------------
-#
-#   Copyright 2018-2019 Fetch.AI Limited
-#
-#   Licensed under the Apache License, Version 2.0 (the "License");
-#   you may not use this file except in compliance with the License.
-#   You may obtain a copy of the License at
-#
-#       http://www.apache.org/licenses/LICENSE-2.0
-#
-#   Unless required by applicable law or agreed to in writing, software
-#   distributed under the License is distributed on an "AS IS" BASIS,
-#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#   See the License for the specific language governing permissions and
-#   limitations under the License.
-#
-# ------------------------------------------------------------------------------
-
-"""This test module contains the tests for the OEF communication using an OEF."""
-import time
```

Comment:
Why has this been removed?

Reply:
because similar tests are in `tests/test_channel/test_oef/test_communication.py`.

Reply:
in particular, in the class `TestDefault`.

Reply:
ok didn't see that.